packages/loader/container-loader/src/container.ts

```diff
@@ -278,7 +278,6 @@ export interface IPendingContainerState {
 	 */
 	savedOps: ISequencedDocumentMessage[];
 	url: string;
-	term: number;
```

Comment:
I see that @anthony-murphy marked IPendingContainerState deprecated entirely in `main`, so this isn't really a breaking change (I anticipate we'll stop exporting this interface in `next` as soon as the main-next integrate happens?)